cupy/__init__.py

```diff
@@ -67,7 +67,22 @@ def is_available():
 # =============================================================================
 # Constants (borrowed from NumPy)
 # =============================================================================
+from numpy import Inf  # == None  # NOQA
```

Comment:
Could you remove `# == None`?
(`# == None` is only for `newaxis`, which is None.)